roles/zabbix_agent/tasks/Linux.yml

```
@@ -219,3 +219,10 @@
   tags:
     - init
     - service
+
+- name: "Give zabbix-agent access to system.hw.chassis info"
+  file:
+    path: /sys/firmware/dmi/tables/DMI
+    owner: root
+    group: zabbix
+  when: zabbix_agent_chassis
```

```suggestion
  when: zabbix_agent_chassis | bool
```
It's safer to use bool filter here, as user can pass string value (yes, true,...).